cmd/ui/src/views/Explore/GraphViewV2.tsx

```diff
@@ -14,7 +14,7 @@
 //
```

Comment:
All changes in here are either renaming `useAvailableDomains`, or porting styles from the original `GraphView`